core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/edit-ema-palette/components/edit-ema-palette-contentlets/edit-ema-palette-contentlets.component.scss

```diff
@@ -46,6 +46,7 @@
     background-color: $white;
     cursor: grab;
     margin: 0 $spacing-3;
+    z-index: 1;
```

Comment:
When we drag an item from the palette and this item is covered by some other component due to overflow, we can see the drag image cut by that component. Setting a z-index as 1 fix this problem. You will see this same line in the content-type component